.github/workflows/unit_tests.yml

```diff
@@ -1,3 +1,5 @@
+# Contains jobs corresponding to unit tests. Integration (e.g. emulator or screenshot testing) or End-to-End tests should not be triggered here.
+
 name: Unit Tests (Robolectric - Bazel)
```

Comment:
Can we name this file `.github/workflows/bazel_unit_tests.yml`

Reply:
Right now the name is indicating Robolectric & Bazel, but once we remove Gradle this is really just a place to hold unit tests (which is actually the intent). Bazel is a bit irrelevant in that way.

I'm fine with renaming 'main' to be 'gradle_checks' or similar, though, if that would help clarify. WDYT?